aws/resource_aws_s3_bucket_test.go

```diff
@@ -755,7 +755,7 @@ func TestAccAWSS3Bucket_Logging(t *testing.T) {
 	})
 }
 
-func TestAccAWSS3Bucket_Lifecycle(t *testing.T) {
+func TestAccAWSS3Bucket_LifecycleBasic(t *testing.T) {
```

Comment:
Note for reviewers: this allows to target this acceptance test without also matching the `TestAccAWSS3Bucket_LifecycleExpireMarkerOnly` test.